scripts/custom_yocto.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+''' Script for building custom bitbake target '''
+import os
+import sys
+import argparse
+import yaml
+
+
+def call_bitbake(yocto_dir, conf):
+    ''' Call bitbake '''
+    cmd = [
+        f"cd {yocto_dir}",
+        f'. ./poky/oe-init-build-env {conf["dir"]}',
+        f'bitbake {conf["target"]}',
+    ]
+    line = " && ".join(c for c in cmd)
+    ret = os.system(f'bash -c "{line}"')
+    return ret >> 8
```

Comment:
Why it is shifted?

Reply:
LSB has signal. MSB has return code of subprocess